tests/e2e/Services/Projects/ProjectsConsoleClientTest.php

```diff
@@ -4118,6 +4123,18 @@ public function testUpdateVariable(array $data)
         $this->assertEquals("APP_TEST_UPDATE_1", $variable['body']['key']);
         $this->assertEmpty($variable['body']['value']);
 
+        $response = $this->client->call(Client::METHOD_PUT, '/project/variables/' . $data['secretVariableId'], array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $data['projectId'],
+            'x-appwrite-mode' => 'admin',
+        ], $this->getHeaders()), [
+            'key' => 'APP_TEST_UPDATE_1',
+            'secret' => false,
+        ]);
+
+        $this->assertEquals(400, $response['headers']['status-code']);
+        $this->assertEquals('Secret variables cannot be marked as non-secret. Please re-create the variable if this is your intention.', $response['body']['message']);
```

Comment:
Message often change, lets remove from test. Checking status code is good enough